src/txmempool.cpp

```diff
@@ -82,18 +83,19 @@ CTxMemPoolEntry::CTxMemPoolEntry(const CTransactionRef& tx, CAmount fee,
       entryHeight{entry_height},
       spendsCoinbase{spends_coinbase},
       sigOpCost{sigops_cost},
+      m_modified_fee{nFee},
       lockPoints{lp},
       nSizeWithDescendants{GetTxSize()},
       nModFeesWithDescendants{nFee},
       nSizeWithAncestors{GetTxSize()},
       nModFeesWithAncestors{nFee},
       nSigOpCostWithAncestors{sigOpCost} {}
 
-void CTxMemPoolEntry::UpdateFeeDelta(CAmount newFeeDelta)
+void CTxMemPoolEntry::UpdateModifiedFee(CAmount fee_diff)
```

Comment:
Would a better name be `IncrementModifiedFee`? The verb "update" isn't wrong, but it could mean simply setting a variable to a value (as in a simple assignment statement). The verb "increment" makes it clear that a variable, or, in this case, 3 variables, are being incremented by the argument. It's true that the previous name used the verb "update", so maybe it would be best to preserve that. But I just thought I'd mention it for consideration.

Reply:
I agree, but maybe this can be done as a follow up to also rename the `Update*State` functions, which do the same thing, but are not touched in this pull request.